sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -80,14 +80,9 @@ export class StreamingReceiver extends MessageReceiver {
 
   /**
    * Starts the receiver by establishing an AMQP session and an AMQP receiver link on the session.
-   *
-   * @param {OnMessage} onMessage The message handler to receive servicebus messages.
-   * @param {OnError} onError The error handler to receive an error that occurs while receivin messages.
-   */
-  receive(onMessage: OnMessage, onError: OnError): void {
```

Comment:
Is this a breaking change since we are removing 2 parameters from this method?

The tests are failing -- is that related to this?

Reply:
The test failures on CI seem to be browser related.

This isn't a breaking change as the change is for an internal method. This method is no longer being exposed and sits behind the actual user facing 'registerMessageHandler()` method

On a side note, I forgot adding one file to the commit from local, fixed it - running the tests on Node in local and they are passing so far. 
The browser related test failures need a closer look and will be unrelated.